dream-num/univer
# fix(formula): fix progress bar not loaded

- [x] Remove `IProgressService`
- [x] Replace `deserializeRangeWithSheet` with `deserializeRangeWithSheetWithCache`
- [x] add config `intervalCount` for formula engine. The formula calculation quantity interval for waiting for the main thread message in the worker. Each time the formula calculates the `intervalCount` quantity, it will receive a main thread message to support stopping the calculation. Default is 500.

## How to test?

Import a table with 500,000 formulas and check if a progress bar appears (it will take about 10 seconds to calculate)






## Pull Request Checklist
